### PR TITLE
Update to engine v1.71

### PR DESCRIPTION
**Pull request by slimbuck**
* update dependencies, including latest engine
* update manifest to get pwa working

### CODE REVIEW

**Review by mvaligursky — APPROVED** [review]

File: static/manifest.json [hunk 1]
```diff
@@ -1,18 +1,15 @@
 {
+    "name": "PlayCanvas glTF Viewer",
+    "short_name": "glTF Viewer",
+    "description": "Fast and lightweight glTF and gaussian splat PLY viewer.",
+    "display": "standalone",
+    "start_url": "/model-viewer/",
```
Comment (willeastcott) [inline static/manifest.json:6]: Isn't `viewer` the preferred URL? We actually support both (without redirecting). I'm thinking we should pick one and redirect from the other.

File: package.json [hunk 2]
```diff
@@ -60,28 +60,28 @@
...
+    "rollup": "^4.18.0",
     "rollup-plugin-copy": "^3.5.0",
     "rollup-plugin-sass": "^1.12.22",
     "serve": "^14.2.3",
     "tslib": "^2.6.2",
     "typescript": "^5.4.5"
   },
   "optionalDependencies": {
-    "@rollup/rollup-linux-x64-gnu": "4.12.0"
+    "@rollup/rollup-linux-x64-gnu": "4.18.0"
```
[the remark this answers is not in the document]
Comment (slimbuck) [reply package.json]: Removed, thanks

File: static/manifest.json (hunk 1 above)
Comment (slimbuck) [reply static/manifest.json:6]: I've always thought `viewer` is just too generic. I also prefer to have the github repo name match the application name match the url. Model-viewer for me strikes the balance between being general enough without being too specific. I will concede though that written out it looks a little clunky, so it's your call :)

While we're at it, the repo's short name above is `glTF Viewer`, which is also not quite correct.

**Review by willeastcott — APPROVED** [review]

File: static/manifest.json (hunk 1 above)
Comment (slimbuck) [reply static/manifest.json]: Renamed application to "Model Viewer" and using path /viewer/.

**Pull request merged**